tests/reducers/settingEncounterRoleReducer.test.js

```diff
@@ -0,0 +1,52 @@
+import deepFreeze from 'deep-freeze';
+import {
+    settingEncounterRoleSuccess,
+    settingEncounterRoleLoading,
+    settingEncounterRoleFailure,
+} from '../../app/js/actions/settingEncounterRole';
+import settingEncounterRoleReducer from '../../app/js/reducers/settingEncounterRoleReducer';
+
+
+describe('Setting Encounter Tole reducer', () => {
+    let initialState;
+    beforeEach(() => {
+        initialState = {
+            settingEncounterRole: '',
+            isLoading: false,
+            roleError: '',
+        };
+        deepFreeze(initialState);
```

Comment:
What does this do?

Reply:
deep-freeze is used to ensure that the initialState is immutable in the redux-store.

Reply:
okay